dappnode_package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hopr.public.dappnode.eth",
-  "version": "1.0.5",
```

Comment:
Shouldn't this be a patch release update?

Reply:
Unfortunately, the dappnodesdk tool complains if I do 1.1.0